lib/lfrfid/tools/bit_lib.h

```diff
@@ -141,6 +149,14 @@ void bit_lib_copy_bits(
     const uint8_t* source,
     size_t source_position);
 
+/**
+ * @brief Reverse bits in a byte
+ * 
+ * @param byte Byte
+ * @return uint8_t the reversed byte
+ */
+uint8_t bit_lib_reverse_8(uint8_t byte);
```

Comment:
bit_lib_reverse_8_fast, and move to bit_lib_reverse_16_fast

Reply:
Done